packages/ai-api/src/utils/deployment-resolver.ts

```diff
@@ -206,7 +206,7 @@ export async function getAllDeployments(
  * @returns The ID of the deployment, if found.
  * @internal
  */
-export async function getDeploymentId(
+export async function getFoundationModelDeploymentId(
```

Comment:
[req] I think @ZhongpinWang 's suggestion was to move the respective `get...DeploymentId` methods to their respective packages. So `getFoundationModelDeploymentId` to the foundation-model package. And Orchestration to its own.